### DISCUSSION TITLE
[Feature] Server status endpoint

### DISCUSSION THREAD

**Discussion by SiebeVE** [immich-app/immich#11446] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When using uptimerobot ([uptimerobot.com](https://uptimerobot.com/)) it right now reports the immich instance as Down because the HEAD request returns a 404:
```bash
➜   curl -i -I https://my-instance
HTTP/2 404
content-type: application/json; charset=utf-8
```

It would be nice if this can be changed so I can monitor the uptime of my instance.
It would be even cooler if there is a dedicated endpoint that reflects the status of the server: `/api/server/status` which returns a 200 on a HEAD request and some json with properties like; database, worker,... to see if they are up.
We could then protect that link with a secret so only we can access that publicly available link

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by eKristensen** [comment]
Just use [Your Immich instance]/api/server-info/ping

Works for me on the free Uptime Robot plan.

I found it here: https://www.reddit.com/r/immich/comments/1csqr19/heartbeathealthcheck_url/
